test/Microsoft.TestPlatform.AcceptanceTests/ExecutionTests.cs

```diff
@@ -16,35 +16,92 @@ namespace Microsoft.TestPlatform.AcceptanceTests;
+        ExitCodeEquals(1); // failing tests
+        StdErrHasTestRunFailedMessageButNoOtherError();
+        StdOutHasNoWarnings();
+    }
+
+    [TestMethod]
+    [TestCategory("Windows-Review")]
+    [TestPlatformCompatibilityDataSource()]
+    //[TestPlatformCompatibilityDataSource(BeforeFeature = Features.ATTACH_DEBUGGER, AfterAdapterFeature = Features.MSTEST_IFRAMEWORK_HANDLE_99)]
+
+    public void RunTestsFromMultipleMSTestAssemblies(RunnerInfo runnerInfo)
+    {
+        SetTestEnvironment(_testEnvironment, runnerInfo);
+
+        var assemblyPaths = BuildMultipleAssemblyPath("MSTestProject1.dll", "MSTestProject2.dll");
+
+        InvokeVsTestForExecution(assemblyPaths, testAdapterPath: null, FrameworkArgValue, string.Empty);
+
+        ValidateSummaryStatus(passed: 2, failed: 2, skipped: 2);
+        ExitCodeEquals(1); // failing tests
+        StdErrHasTestRunFailedMessageButNoOtherError();
+        StdOutHasNoWarnings();
+    }
+
+    [TestMethod]
+    [TestCategory("Windows-Review")]
+    [TestHostCompatibilityDataSource]
+    public void RunMultipleMSTestAssembliesOnVstestConsoleAndTesthostCombinations(RunnerInfo runnerInfo)
+    {
+        SetTestEnvironment(_testEnvironment, runnerInfo);
+
+        var assemblyPaths = BuildMultipleAssemblyPath("MSTestProject1.dll", "MSTestProject2.dll");
+
+        InvokeVsTestForExecution(assemblyPaths, testAdapterPath: null, FrameworkArgValue, string.Empty);
+
+        ValidateSummaryStatus(2, 2, 2);
+        ExitCodeEquals(1); // failing tests
+    }
+
+
+    [TestMethod]
+    [TestCategory("Windows-Review")]
+    [RunnerCompatibilityDataSource]
+    public void RunMultipleMSTestAssembliesOnVstestConsoleAndTesthostCombinations2(RunnerInfo runnerInfo)
+    {
+        SetTestEnvironment(_testEnvironment, runnerInfo);
+
+        var assemblyPaths = BuildMultipleAssemblyPath("MSTestProject1.dll", "MSTestProject2.dll");
 
-        InvokeVsTestForExecution(assemblyPaths, GetTestAdapterPath(), FrameworkArgValue, string.Empty);
+        InvokeVsTestForExecution(assemblyPaths, testAdapterPath: null, FrameworkArgValue, string.Empty);
 
         ValidateSummaryStatus(2, 2, 2);
         ExitCodeEquals(1); // failing tests
     }
 
+
+    // TODO: This one mixes different frameworks, I can make it work, but it is worth it? We are going to test
+    // the two respective versions together (e.g. latest xunit and latest mstest), but does using two different test
+    // frameworks have any added value over using 2 mstest dlls?
     [TestMethod]
     [NetFullTargetFrameworkDataSource(inIsolation: true, inProcess: true)]
     [NetCoreTargetFrameworkDataSource]
     public void RunMultipleTestAssembliesWithoutTestAdapterPath(RunnerInfo runnerInfo)
     {
         SetTestEnvironment(_testEnvironment, runnerInfo);
 
-        var assemblyPaths = BuildMultipleAssemblyPath("SimpleTestProject.dll").Trim('\"');
+        var assemblyPaths = BuildMultipleAssemblyPath("SimpleTestProject.dll");
```

Comment:
You are right. Not my code though. I reverted a lot of changes to these methods because they were cosmetic and polluted this PR. Unlike the trim that actually breaks stuff if you put it back.

Reply:
Closing this. I have opened a task ticket for this cleanup.